bbn-test-4/finality-providers/README.md

```diff
@@ -71,13 +71,18 @@ The deposit will be locked for `52560` blocks (i.e. ~one year),
 and will not be counted towards the active stake of the system.
 Note that the deposit is still fully in the custody of the creator of the
 transaction, but will only become unlocked after the deposit period expires.
+The deposit amount of `10000000` signet satoshi is the minimum amount required
```

Comment:
Let's highlight this with a warning etc?

Reply:
like this? https://github.com/babylonchain/networks/pull/193/commits/85c62fa0c8cbfa062b7b8cce14636dd841f39571